todo-app/src/Components/InputField.js

```diff
@@ -0,0 +1,17 @@
+//jshint esnext: true
+
+import React, { Component } from "react";
+
+export default class InputField extends Component {
+  render() {
+    return (
+      <input
+        type={this.props.type}
+        placeholder={this.props.placeholder}
+        value={this.props.value}
+        onChange={this.props.handleChange}
```

Comment:
Small thing to think about: the component `InputField` is there to "wrap" the `<input/>` tag right? That is, if I use `<InputField>` instead of `<input>` I don't have to worry about the DOM.

That's great, but as you're passing `this.props.handleChange` to `onChange`, you expect the code that uses this component to do `event.target.value`, which is DOM-specific.

I would choose to say: `InputField` has a prop `onChange` that takes a callback which will just receive the changed value, and then use it like this: `onChange={event => { this.props.onChange(event.target.value) }}`.

Now if I *use* this component, I can write:

```
<InputField
  value={this.state.myValue}
  onChange={value => this.setState({myValue: value})}
/>
```

See? No more references to DOM-specific stuff!